lib/presentation/pages/layouts/profile/profile_image.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class ProfileImage extends StatelessWidget {
+  final String image;
+  final String avatar;
+  const ProfileImage({super.key, required this.image, required this.avatar});
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipOval(
+      child: SizedBox(
+        height: 128,
+        width: 128,
```

Comment:
Try using AspectRatio widget wherever possible instead of explicitly mentioning width and height. 
aspectRatio = width / height